source/apiVolontaria/volunteer/tests/tests_model_Event.py

```diff
@@ -447,3 +448,22 @@ def test_is_started_property_true(self):
         )
 
         self.assertEqual(event.is_started, True)
+
+    def test_duration(self):
+        """
+        Ensure we have True if the event is started
+        """
+        start_date = timezone.now()
+        end_date = start_date + timezone.timedelta(
+            minutes=100
+        )
+
+        event = Event.objects.create(
+            cell=self.cell,
+            cycle=self.cycle,
+            task_type=self.task_type,
+            start_date=start_date,
+            end_date=end_date,
+        )
+
+        self.assertEqual(round(event.duration, 2), Decimal('1.67'))
```

Comment:
Please test directly `event.duration` and not the rounded value. Actually your test not really test the value returned.

Reply:
I'm not sure how to test a value that has several decimal like 1.666666666666667...  Do you have a suggestion ?  Rounding the value was getting me there and the value is fixed since it's always the same gap of time

Reply:
Their are multiple solution :

1. You can directly calculate the value in the test
2. You can use other data of test to check a more easy value
3. Maybe it's not good to have an epic value like this one and the `@property` need to round itself.